### PR TITLE
Fix various typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S "./docs/ChangeLog.*" -L aswell,ba,gord,ifset,iif,inout,invokation,lod,msdos,pard,reenable,ro,siz,tesselator`

### CODE REVIEW

File: include/Inventor/SbByteBuffer.h [hunk 1]
```diff
@@ -73,7 +73,7 @@ class COIN_DLL_API SbByteBuffer {
 
 #endif // !COIN_SBBYTEBUFFER_H
 
-//The SBBYTEBUFFER_PRIVATE_VARIABLES must survice an inclusion from the .icc file
+//The SBBYTEBUFFER_PRIVATE_VARIABLES must service an inclusion from the .icc file
```
Comment (VolkerEnderlein) [inline include/Inventor/SbByteBuffer.h:76]: I guess this should rather read `survive` than `service`.

**Comment by VolkerEnderlein** [comment]
Thanks for fixing the typos and providing the PR. Please, see attached comments to files.

File: include/Inventor/SbByteBuffer.h (hunk 1 above)
Comment (luzpaz) [reply include/Inventor/SbByteBuffer.h]: Corrected in [759bacd](https://github.com/coin3d/coin/pull/486/commits/759bacdeef2e4ab87dd1d8a9d86c9ce7c6fcb662)

**Comment by luzpaz** [comment]
3 more typos found, please review [680d8f5](https://github.com/coin3d/coin/pull/486/commits/680d8f57f231afc93e5930db8effdee853293a6d)

File: src/nodes/SoPerspectiveCamera.cpp [hunk 2]
```diff
@@ -85,7 +85,7 @@ SoPerspectiveCamera::SoPerspectiveCamera()
 {
   SO_NODE_INTERNAL_CONSTRUCTOR(SoPerspectiveCamera);
 
-  SO_NODE_ADD_FIELD(heightAngle, (float(M_PI)/4.0f));  // 45�.
+  SO_NODE_ADD_FIELD(heightAngle, (float(M_PI)/4.0f));  // 45�.
```
[the remark this answers is not in the document]
Comment (luzpaz) [reply src/nodes/SoPerspectiveCamera.cpp]: [643d276](https://github.com/coin3d/coin/pull/486/commits/643d276b4e9b6994f56e960a53730133a6853b3d)

File: src/nodes/SoPerspectiveCamera.cpp [hunk 3]
```diff
@@ -71,7 +71,7 @@
   \var SoSFFloat SoPerspectiveCamera::heightAngle
 
   The vertical angle of the viewport, also known as "field of view".
-  Default value is 45� (note: value is specified in radians).
+  Default value is 45� (note: value is specified in radians).
```
[the remark this answers is not in the document]
Comment (luzpaz) [reply src/nodes/SoPerspectiveCamera.cpp]: [643d276](https://github.com/coin3d/coin/pull/486/commits/643d276b4e9b6994f56e960a53730133a6853b3d)

**Pull request merged**